pages/web-literacy/colors.js

```diff
@@ -0,0 +1,33 @@
+var categories = require('./categories');
```

Comment:
These colour files are actually new to me, I have not touched them. I'll dig into them a bit see what they are doing and what we can remove.